### ISSUE TITLE
Time remaining is confusing

### ISSUE THREAD

**Issue by rschuetzler** [tomighty/tomighty#86]
The time remaining (in minutes) should round up to the next minute. The only time it's a problem for me is with 1 minute remaining. After it shows 1 minute remaining for a full minute, it then begins the last minute countdown. 1 minute remaining should never be shown if there is a 60 second countdown.

**Comment by ccidral** [comment]
Makes sense. Thanks for the feedback, I will make this change available in the next version.
